documentation/general/design/dash-sonic-hld.md

```diff
@@ -339,8 +339,23 @@ metering_bucket          = bucket_id                 ; metering and counter
 A high-level module interaction is captured in the following diagram.
 
   ![dash-high-level-diagram](./images/hld/dash-high-level-design.svg)
-  
-### 3.3.1 SONiC host containers
+
+
+### 3.3.1 DASH Schema Relationships
+The [figure below](#schema_relationships) illustrates the various schema and their transformations into the various SONiC layers, including:
+* gNMI northbound API, which uses YANG to specify schema
+* Redis APP_DB and DASH_APP_DB, which use [ABNF](https://github.com/Azure/SONiC/blob/master/doc/mgmt/Management%20Framework.md#12-design-overview) schema definition language. Redis objects can be directly manipulated using [SAI-redis](https://github.com/Azure/sonic-sairedis) clients.
```

Comment:
Based on PR #118, we are re-using APP_DB and each objects shall be specified with DASH_ prefix.  Can you please update? This can be merged on top of PR 118.

Reply:
I made the changes to text & figure. Following PR118 merge, I will sync this and we can merge it.